rpc/block_svc_routines.c

```diff
@@ -258,6 +260,67 @@ blockStr2arrayAddToJsonObj(json_object *json_obj, char *string, char *label)
 }
 
 
+static int
+blockCheckBlockLoadedStatus(char *block_name, char *gbid, blockResponse *reply)
+{
+
+  int ret = -1;
+  char *exec = NULL;
+  int is_loaded = true;
+
+
+  if (GB_ASPRINTF(&exec, GB_TGCLI_CHECK, block_name, gbid) == -1) {
+    goto out;
+  }
+
+  ret = gbRunner(exec);
+  if (ret == -1) {
+    GB_ASPRINTF(&reply->out, "command exit abnormally for %s\n", block_name);
+    LOG("mgmt", GB_LOG_ERROR, "%s", reply->out);
+    goto out;
+  } else if (ret == 1) {
```

Comment:
exit value 0 means success and non-zero is failure. So are we sure that ret value is going to be 1 for this failure? Same comment for the next set of if/else as well

Reply:
Yes, it will be 1 (not something newly introduced)